src/main/java/io/jenkins/plugins/coverage/targets/Ratio.java

```diff
@@ -150,18 +150,18 @@ public int hashCode() {
      * @return the ratio
      */
     public static Ratio create(float x, float y) {
-        int xx = (int) x;
-        int yy = (int) y;
-
-        if (xx == x && yy == y) {
-            int idx = yy * (yy + 1) / 2 + xx;
-            if (0 <= idx && idx < COMMON_INSTANCES.length) {
-                Ratio r = COMMON_INSTANCES[idx];
-                if (r == null)
-                    COMMON_INSTANCES[idx] = r = new Ratio(x, y);
-                return r;
-            }
-        }
+//        int xx = (int) x;
```

Comment:
the code should be removed OR there should be a comment explaining why it's commented out